package/MDAnalysis/analysis/hole2/hole.py

```diff
@@ -915,7 +915,7 @@ def create_vmd_surface(self, filename='hole.vmd', dot_density=15,
                 pass
 
             tri = '{ { ' + ' } { '.join(list(map(' '.join, shapes))) + ' } }'
-            frames.append('set triangles({i}) '.format(i=i) + tri)
+            frames.append(f'set triangles({frame}) ' + tri)
```

Comment:
Need to index with `i` for VMD, see https://github.com/MDAnalysis/mdanalysis/pull/3489#issuecomment-1003202993.

Reply:
Does it make sense to just do that, output N "consecutive frames" (which may have been picked non-consecutively) and let users deal with the mismatch?

Reply:
The only way that VMD will "play" the surface movie is by writing consecutive frames. It's the user's responsibility to write the appropriate trajectory as in my example in the comment.